### ISSUE TITLE
Expose accessibility configuration on view props

### ISSUE THREAD

**Issue by MaxDesiatov** [MaxDesiatov/Tokamak#39]
Accessibility [labels, hints and traits](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW5) should be available as Props on all components that render to views
